dashboard/src/components/partials/SilencesList/SilencesList.js

```diff
@@ -62,25 +62,20 @@ class SilencesList extends React.Component {
       }
 
       ${Pagination.fragments.pageInfo}
+      ${ClearSilencesDialog.fragments.silence}
       ${ListItem.fragments.silence}
     `,
   };
 
   state = {
     silence: null,
+    openDialog: false,
```

Comment:
might be nice if this were a bit more descriptive, maybe `openClearDialog`?

Reply:
Updated